src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx

```diff
@@ -228,24 +230,21 @@ RxAmplify.Auth.signIn("username", "password", options).subscribe(
                     // `signIn` api to trigger signIn flow again.
                     break;
                 }
-                case CONFIRM_SIGN_UP: {
-                    Log.i("AuthQuickstart", "Confirm signup, additional info: " + nextStep.getAdditionalInfo());
-                    // User was not confirmed during the signup process.
-                    // Invoke `confirmSignUp` api to confirm the user if
-                    // they have the confirmation code. If they do not have the
-                    // confirmation code, invoke `resendSignUpCode` to send the
-                    // code again.
-                    // After the user is confirmed, invoke the `signIn` api again.
-                    break;
-                }
                 case DONE: {
                     Log.i("AuthQuickstart", "SignIn complete");
                     // User has successfully signed in to the app
                     break;
                 }
             }
         },
-        error -> Log.e("AuthQuickstart", "SignIn failed: " + error)
+        error -> {
+            if (error instanceof UserNotConfirmedException) {
+            Log.e("AuthQuickstart", "Failed to sign in due to user not being confirmed " + error)
+            }
```

Comment:
Else should be on the same line as the closing if brace.

Reply:
yeah the indentation got removed for some reason. Going to fix it